src/olympia/lib/settings_base.py

```diff
@@ -1175,6 +1175,7 @@ def read_only_mode(env):
 ES_TIMEOUT = 30
 ES_DEFAULT_NUM_REPLICAS = 2
 ES_DEFAULT_NUM_SHARDS = 5
+ES_COMPRESS = True
```

Comment:
Per the description, it might be better to have the default be Off, and turn it on in specific environments? Is that not possible currently?

Reply:
We could but I expect to turn it on by default very soon or completely revert - so I figured might as well enable it by default, rather that having to enable it in local/dev/stage.

Reply:
Fair enough.